chroma-common/blockdevices/blockdevice.py

```diff
@@ -99,6 +99,10 @@ def device_path(self):
     def initialise_driver(cls, managed_mode):
         return None
 
+    @classmethod
+    def terminate_driver(cls):
+        return None
```

Comment:
both are equivalent technically since an empty return is the same as return None